theme/snippets/cart-item.liquid

```diff
@@ -8,8 +8,17 @@
     <div class="CartItem__image-container flex items-center justify-center absolute t0 r0 h100 w100">
       <img class="CartItem__image w100 h100 fit-contain xs:py1" src={{ item | img_url: 'medium' }}" alt="{{ item.title | escape }}">
     </div>
-    <div class="CartItem__details absolute r0 b0 p_625 w100">
+    <div class="CartItem__details absolute r0 b0 p_625 w100 flex justify-between">
       <button data-line-item-variant-id={{item.variant_id}} data-line-item-quantity={{item.quantity}} data-remove-from-cart-button aria-label={{ 'snippets.cart.remove_button.aria_label' | t }} class="button--style-no-style sans-xs color-white uppercase">{{ 'snippets.cart.remove_button.text' | t }} </button>
+      <div 
+        data-quantity-selector 
+        data-quantity-selector-variant-id="{{ item.variant_id }}"
+        data-quantity="{{ item.quantity }}"
+        class="CartItem__quantity-spinner serif flex flex-row">
+        <button data-decrement class="serif mr1 md:mr_25 color-white button--style-no-style">–</button>
+        <span data-quantity-indicator class="color-white">{{ item.quantity }}</span>
+        <button data-increment {% if item.quantity == item.variant.inventory_quantity %} disabled {% endif %} class="serif ml1 md:ml_25 color-white button--style-no-style">+</button>
+        </div>
```

Comment:
`{% if item.quantity == item.variant.inventory_quantity %} disabled {% endif %}`

What do you think @sepowitz ? The item already shows as "Sold out"/disabled add to cart button if it's not available, so I think it may not be able to get added to cart if it's not available.

Reply:
Not exactly sure what your questions is, but disabling this button would be for the case that say a single quantity of an item was available when the user goes to a product page. They add that product to cart, and then in their cart we disable the increment button, as there is no more inventory of that item available. Does that make sense?

Also, I think we can do 
```
{% if not item.variant.available %} disabled {% endif %}
```

Reply:
if in cart, button says add to cart
if not in cart, button says add to cart or sold out 
prevent decrementing past 1
disable the decrement button if the item.quantity = 1
they can click remove to remove it

if disabled (in css) cursor = not allowed